.github/workflows/coding-standards.yml

```diff
@@ -94,6 +94,9 @@ jobs:
       - name: Check test suite files for warnings
         run: phpcs tests -q --report=checkstyle | cs2pr
 
+      - name: Ensure version-controlled files are not modified during the tests
```

Comment:
Instead of checking every workflow for changes, would it make more sense to have a singular workflow that runs the commands that would potentially generate changes to versioned files and checks for local changes?

I'm also not sure if this return will negatively effect the annotations added here.

Reply:
I implemented the catch-all so there's no need to worry about which tests could modify files and which don't.

Reply:
Currently looking into why `script-loader-packages.php` gets deleted during the test-npm workflow.

Reply:
It's probably because of `grunt clean` being tested.

Reply:
You should be able to add the diff check prior to `grunt clean` being checked and it would check for changes before the file gets deleted.

It won't be the very last item running, but I think that is fine there.

Reply:
Discussed this with Jon, decided to fix the cleaned paths so script-loader-packages.php doesn't get deleted. Confirmed that the build still behaves as expected.